src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/rest/dto/ListingGetDto.java

```diff
@@ -0,0 +1,133 @@
+package ch.uzh.ifi.fs22.sel.group15.cozycave.server.rest.dto;
+
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.constant.Gender;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.constant.ListingType;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.Location;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.Picture;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.user.User;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+
+public class ListingGetDto implements Serializable {
```

Comment:
This I don't quite understand

Reply:
you have to change some stuff here too (they have to be the same so that the mapper can properly work, except additional or removed stuff, but e.g. the types must match) because you changed it in the Listing entity (entity/Listing.java)